pkg/agent/supportbundlecollection/support_bundle_controller.go

```diff
@@ -299,13 +299,26 @@ func (c *SupportBundleController) uploadSupportBundle(supportBundle *cpv1b2.Supp
 	if _, err := outputFile.Seek(0, 0); err != nil {
 		return fmt.Errorf("failed to upload to the file server while setting offset: %v", err)
 	}
+	// #nosec G106: users should set hostPublicKey, accepting arbitrary keys is not recommended.
+	hostKeyCallback := ssh.InsecureIgnoreHostKey()
```

Comment:
Do you think it's time to make this part of code in a common package? It seems pretty duplicated after added this.

Reply:
I did some refactoring. There may be opportunities for more in the future.